From 41ec46d8ba2c8c7aafd9bc4916f681a270449521 Mon Sep 17 00:00:00 2001
From: "Juan F. Codagnone" <juan@leak.com.ar>
Date: Thu, 3 Jun 2021 12:44:39 -0300
Subject: [PATCH] fix

---
 src/stm.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/stm.h b/src/stm.h
index f20a68f..b2822a6 100644
--- a/src/stm.h
+++ b/src/stm.h
@@ -1,6 +1,8 @@
 #ifndef STM_H_wL7YxN65ZHqKGvCPrNbPtMJgL8B
 #define STM_H_wL7YxN65ZHqKGvCPrNbPtMJgL8B
 
+#include "selector.h"
+
 /**
  * stm.c - pequeño motor de maquina de estados donde los eventos son los
  *         del selector.c
@@ -34,8 +36,6 @@ struct state_machine {
     const struct state_definition *current;
 };
 
-struct selector_key *key;
-
 /**
  * definición de un estado de la máquina de estados
  */
-- 
2.38.1